app/src/main/java/com/movies/data/local/database/FavouriteMovieDatabase.kt

```diff
@@ -0,0 +1,11 @@
+package com.movies.data.local.database
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import com.movies.data.local.dao.FavouriteMoviesDao
+import com.movies.data.local.entity.FavouriteMovieEntity
+
+@Database(entities = [FavouriteMovieEntity::class], version = 1)
+abstract class FavouriteMovieDatabase : RoomDatabase() {
+    abstract fun favouriteMovieDao(): FavouriteMoviesDao
+}
```

Comment:
you can move version to constant, for example : 
```suggestion
@Database(entities = [FavouriteMovieEntity::class], version = DB_VERSION )
abstract class FavouriteMovieDatabase : RoomDatabase() {
    abstract fun favouriteMovieDao(): FavouriteMoviesDao
}
companion object {
        const val DB_VERSION = 1
    }
```

Reply:
I prefer in this way :3

Reply:
why then you use constants for base url ?

Reply:
you can paste whole string 😐

Reply:
it's ugly there but not here